README.md

```diff
@@ -108,3 +104,27 @@ jobs:
 If you need to use an in-development version of these tools from a branch, you can update the `ref` in `repositories` section to point to a different branch.
 
 The `build-pipeline.yml` template has a few parameters. You can find out more about them in [build-pipeline.yml](https://github.com/shotgunsoftware/tk-ci-tools/blob/master/build-pipeline.yml).
+
+### Create a build in Azure Pipelines
+
+Once you've added your `azure-pipelines.yml` file to your repository, it's time to add the repository to the Azure Pipelines web service.
+
+1. Go to the [Shotgun Ecosystem Azure Pipelines](https://dev.azure.com/shotgun-ecosystem/Toolkit/_build) page.
+2. Select any pipeline from the list and clone it.
+   ![cloning_build](resources/cloning_build.png)
+3. In the YAML tab, update the build info
+3.1 The name can be updated from the right-hand side of the page. You must name the build after the name of the repository.
```

Comment:
Github's markdown isn't parsing this correctly:

<img width="923" alt="Screen Shot 2019-11-04 at 11 43 10 AM" src="https://user-images.githubusercontent.com/5922577/68152367-5c931b80-fef8-11e9-89d3-59901d98b51e.png">